### PR TITLE
fix the docker build

### PR DESCRIPTION
**Pull request by vnepogodin**
use source from github repo

solving ticket https://github.com/AppFlowy-IO/appflowy/issues/13

### CODE REVIEW

**Comment by appflowy** [comment]
Thanks for fixing docker issues.

**Comment by cata0309** [comment]
Why did you merged it? Did you read the comments I wrote?

**Comment by appflowy** [comment]
@cata0309 Thank you for reviewing this change. I have reverted it. 

@vnepogodin Would you mind considering @cata0309 advice?

**Comment by vnepogodin** [comment]
Well I will mind the @cata0309 advice

File: backend/Dockerfile [hunk 1]
```diff
@@ -1,16 +1,23 @@
 
-FROM rust:1.53.0
```
[the remark this answers is not in the document]
Comment (vnepogodin) [reply backend/Dockerfile:2]: I will use the hardcoded version then

File: backend/Dockerfile [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
+FROM rust:latest
 # Let's switch our working directory to `app` (equivalent to `cd app`)
 # The `app` folder will be created for us by Docker in case it does not
 # exist already.
 WORKDIR /app
 
-COPY . .
+RUN apt-get update -y \
```
[the remark this answers is not in the document]
Comment (vnepogodin) [reply backend/Dockerfile:8]: alpine one is getting linker error:
```
error: linking with `cc` failed: exit status: 1
  |
  = note: "cc" "-m64" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.0.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.1.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.10.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.11.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.12.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.13.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.14.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.15.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.2.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.3.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.4.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.5.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.6.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.7.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.8.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.paste.efa3acef-cgu.9.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.4up353lxtag6m3sj.rcgu.o" "/app/backend/target/release/deps/paste-c0a7d59cb15f68dd.2gw3vv2rzqg2vtpz.rcgu.o" "-Wl,--as-needed" "-L" "/app/backend/target/release/deps" "-L" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib" "-Wl,-Bstatic" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/libproc_macro-9c67d9d80d535a07.rlib" "-Wl,--start-group" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/libstd-d38ce8ab347f7cec.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/libpanic_unwind-dd5634a44e0bc737.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/libminiz_oxide-548f188d638ec622.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/libadler-2939cf007aeaa475.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/libobject-0f834c3278ab1bc1.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/libmemchr-ab431e63115b4770.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/libaddr2line-5ff1cd58432815a7.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/libgimli-c4bcb9cc2caf9a31.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/libstd_detect-a6671c80e63c8252.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/librustc_demangle-54f4c4539400ec23.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/libhashbrown-410dd351b2789634.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/librustc_std_workspace_alloc-55728ce497e9118a.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/libunwind-9adf642fb7be890a.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/libcfg_if-b4470027f3f35d36.rlib" "/tmp/rustc0fMqFL/liblibc-fa9cf38b3026238e.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/liballoc-f6f0abc7cb670bf6.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/librustc_std_workspace_core-4d82127d5995b0ce.rlib" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/libcore-76e8fc619db7bb9b.rlib" "-Wl,--end-group" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib/libcompiler_builtins-2ea647e0d8c6d32c.rlib" "-Wl,-Bdynamic" "-lgcc_s" "-lc" "-Wl,--eh-frame-hdr" "-Wl,-znoexecstack" "-L" "/usr/local/rustup/toolchains/1.56.1-x86_64-unknown-linux-musl/lib/rustlib/x86_64-unknown-linux-musl/lib" "-o" "/app/backend/target/release/deps/libpaste-c0a7d59cb15f68dd.so" "-Wl,--gc-sections" "-shared" "-Wl,-zrelro" "-Wl,-znow" "-nodefaultlibs"
  = note: /usr/lib/gcc/x86_64-alpine-linux-musl/10.3.1/../../../../x86_64-alpine-linux-musl/bin/ld: cannot find crti.o: No such file or directory
          collect2: error: ld returned 1 exit status


   Compiling minimal-lexical v0.2.1
error: could not compile `paste` due to previous error
warning: build failed, waiting for other jobs to finish...
```

[the remark this answers is not in the document]
Comment (cata0309) [reply backend/Dockerfile:8]: Could you share the Dockerfile you are currently working on?

**Pull request merged**